targets/lenovo-x1-carbon.nix

```diff
@@ -273,6 +275,10 @@
       ghaf.development.usb-serial.enable = true;
       ghaf.profiles.debug.enable = true;
     }
+    ../modules/host/secureboot.nix
+    {
+      ghaf.host.secureboot.enable = true;
```

Comment:
For now, I'd set this to `false` by default. Otherwise the mainline X1 builds won't boot before taking the secureboot into use.  Please `git commit --amend` the change and `git push -f` to your PR branch.

Reply:
Sure @vilvo, updated code to make it false by default.